Tests/AutoPilotTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AutoPilotTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+import XCTest
+@testable import KeyboardSupport
+
+class AutoPilotTests: XCTestCase {
+
+    // MARK: - Properties
+    
+    private var viewController: AutoNavigatorViewController! = nil
+    private var window: UIWindow!
```

Comment:
It looks like this property is unused and can be removed.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/59987d774c8357aad3035bc546cc356e5b5a5bcc